osu.Game/Screens/Ranking/ResultsScreen.cs

```diff
@@ -183,6 +195,11 @@ protected override void LoadComplete()
                 api.Queue(req);
 
             statisticsPanel.State.BindValueChanged(onStatisticsStateChanged, true);
+
+            using (BeginDelayedSequence(AccuracyCircle.ACCURACY_TRANSFORM_DELAY + AccuracyCircle.TEXT_APPEAR_DELAY, true))
+            {
+                this.Delay(-1000).Schedule(() => applauseSound?.Play());
```

Comment:
So as suggested I exposed the two variables on `ScorePanel` and made a new public constant on the `AccuracyCircle` called `APPLAUSE_DELAY`, so it's a bit clearer where the delay is coming from, as per your suggestions.

```suggestion
                this.Delay(ScorePanel.RESIZE_DURATION + ScorePanel.TOP_LAYER_EXPAND_DELAY - AccuracyCircle.APPLAUSE_DELAY).Schedule(() => applauseSound?.Play());
```